### PR TITLE
MCR-2167 support scoped IPv6 addresses

### PR DESCRIPTION
**Pull request by yagee-de**
do not parse IP addresses with our code, let it be parsed

[Link to jira](https://mycore.atlassian.net/browse/MCR-2167).

### CODE REVIEW

File: mycore-base/src/main/java/org/mycore/common/MCRSession.java [hunk 1]
```diff
@@ -271,40 +273,21 @@ public String getCurrentIP() {
...
-            ip = newip;
+        try {
+            InetAddress inetAddress = InetAddress.getByName(newip);
+            ip = inetAddress.getHostAddress();
+        } catch (UnknownHostException e) {
+            LOGGER.warn("Exception while parsing new ip {} using old value.", newip, e);
         }
     }
 
+    private static boolean isValidIPAddr(String newip) {
```
Comment (rsteph-de) [inline mycore-base/src/main/java/org/mycore/common/MCRSession.java]: Is it enough, if you just test the first character of the IP?
Does the try-catch (line 279,...) not handle this case as well?

**Review by rsteph-de — APPROVED** [review]
looks good for me

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 11b016a19fff0e792b1ad9e0cdd9b37a0b37311c! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5336-pull_request-935

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 57ecc99e56954f37911e7900fc7818ddfae03f85! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5338-pull_request-935

**Pull request merged**